packages/morph/lib/attr-morph.js

```diff
@@ -6,17 +6,20 @@ function updateProperty(value) {
   this.domHelper.setPropertyStrict(this.element, this.attrName, value);
 }
 
-function updateAttribute(value) {
+function updateAttribute(value, namespace) {
```

Comment:
Please create a separate function for updating a value with a namespace, and assign it to `this._update`. This avoids the branching in logic at update time and ensures that all `_update` functions share an argument signature.

```js
function updateAttributeNS(value) {
  if (value === null) {
    this.domHelper.removeAttribute(this.element, this.attrName);
  } else {
    this.domHelper.setAttributeNS(this.element, this.namespace, this.attrName, value);
  }
}

function AttrMorph(element, attrName, domHelper, namespace) {
  // ....
  this.namespace = namespace || null;
  // ...

  if (element.namespaceURI === svgNamespace || attrName === 'style' || !normalizedAttrName) {
    this.attrName = attrName;
    this._update = updateAttribute;
  } else {
    this.attrName = normalizedAttrName;
    if (this.namespace === null) {
      this._update = updateProperty;
    } else {
      this._update = updateAttributeNS;
    }
  }
}
```